Automattic/pocket-casts-android
# Add screenshot sharing to End of Year

## Description

This PR adds screenshot sharing to the End of Year flow. When screenshot is captured most of animations are settled and paused. I do not pause animations like an infinitely scrolling carousel that do not subtract from the sharing experience. Additionally, when screenshot is being captured auto progress is also paused.

You can skip reviewing fc0c9ff8e1a954523b9f66ad59b6063b6e8fb889 and 0597492b94053591a3e7343783f8d1bd481a227f. As they are commits backporting the Capturable library.

## Testing Instructions

1. Share stories using share button and verify screenshots.
2. Share stories using [the screenshot detection](https://github.com/Automattic/pocket-casts-android/pull/3080) and verify screenshots.
3. Test sharing stories while an animation makes the content invisible.
4. Some apps, like Telegram, support sharing both image and text. Check if these are shared together.

Top 1 Show story is still missing but you can test sharing it as well.

## Screenshots or Screencast 

## Checklist
